index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="main">  
+            <a href="index.html" class="my-logo">My Logo</a>
+        
+            <nav class="my-menu">
+                <div></div>
+                <div></div>
+                <div></div>
+            </nav>
+
+    </header>
+    
+    <section class="headline">
+        <section class="head1">
+            <h1> I’m George
+                Glad to see you!</h1>
+        </section>
+        
+    
+        <p class="primary">I’m a software developer! I can help you
+                 build a product , feature or website Look
+                  through some of my work and 
+                  experience! If you like what you see and
+                   have a project you need coded, don’t 
+                   hestiate to contact me.</p>
+
+        <a href="#"class="connect">LET’S CONNECT </a>
+        
+        <div class="icons" > 
+            <ul class="socials">                
+                <li> <a href="https://twitter.com/geok8376"><img src="img/Vectortwit.png" class="twit"></a> </li>
+                <li> <a href="https://www.linkedin.com/in/george-kinyanjui-030468149/"><img src="img/Linkedin icon.png"  alt="linkedin"> </a>  </li>
+                <li> <a href="#"><img src="img/Vectorm.png" alt="m"></a> </li>
+                <li> <a href="https://github.com/george827"><img src="img/Vectorgit.png" alt="git"></a> </li>
+                <li> <a href="#"><img src="img/Vectorch.png" alt="icon"></a> </li>
+            </ul>
+        </div>
+    </section>
+    
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you remove the **extra spacing** at the right of your page to make it sticks to the Figma design. You can do it by only removing the scrolling at the bottom of the page(i.e. `overflow-x: hidden;`)
#### Here is how your project looks like:
![Screenshot from 2022-09-24 17-39-22](https://user-images.githubusercontent.com/83042742/192106815-4bc3d3c9-5e0b-45cb-9030-4c52503c2b7c.png)